<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentViewWidget</class>
 <widget class="QWidget" name="ContentViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>496</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>&amp;Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="mainTabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Payload</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stack">
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="unsupportedTypePage">
          <layout class="QVBoxLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="unsupportedType">
             <property name="text">
              <string>Unsupported Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="imageViewPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="imageView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="contactViewPage">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="Akonadi::ContactViewer" name="contactView" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="contactGroupViewPage">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="Akonadi::ContactGroupViewer" name="contactGroupView" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="incidenceViewPage">
          <layout class="QVBoxLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="CalendarSupport::IncidenceViewer" name="incidenceView" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mailViewPage">
          <layout class="QGridLayout" name="gridLayout1">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="MessageViewer::Viewer" name="mailView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Attributes</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QTreeView" name="attrView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="attrName">
         <property name="placeholderText">
          <string>New Attribute</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="attrAddButton">
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="attrDeleteButton">
         <property name="text">
          <string>&amp;Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Raw Payload</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="KTextEdit" name="dataView">
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Internals</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Unique ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="id">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Remote ID:</string>
           </property>
           <property name="buddy">
            <cstring>remoteId</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="remoteId"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Revision:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="revision">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="size">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Last modification time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="modificationtime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mimetype:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mimeType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>invalid/invalid</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="KEditListWidget" name="flags">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="buttons">
             <set>KEditListWidget::Add|KEditListWidget::Remove</set>
            </property>
            <property name="title" stdset="0">
             <string>Flags</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Tags</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="KEditListWidget" name="tags">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="buttons">
             <set>KEditListWidget::Add|KEditListWidget::Remove</set>
            </property>
            <property name="title" stdset="0">
             <string>Flags</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>CalendarSupport::IncidenceViewer</class>
   <extends>QWidget</extends>
   <header location="global">CalendarSupport/IncidenceViewer</header>
  </customwidget>
  <customwidget>
   <class>MessageViewer::Viewer</class>
   <extends>QTextBrowser</extends>
   <header location="global">viewer/viewer.h</header>
  </customwidget>
  <customwidget>
   <class>Akonadi::ContactViewer</class>
   <extends>QWidget</extends>
   <header location="global">Akonadi/Contact/ContactViewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Akonadi::ContactGroupViewer</class>
   <extends>QWidget</extends>
   <header location="global">Akonadi/Contact/ContactGroupViewer</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
